File: rest/call-feedback/instance-post-example-1/instance-post-example-1.5.x.php
<?php
// Get the PHP helper library from twilio.com/docs/php/install
require_once '/path/to/vendor/autoload.php'; // Loads the library
use \Twilio\Rest\Client;

// Your Account Sid and Auth Token from twilio.com/user/account
$sid = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
$token = "your_auth_token";
$client = new Client($sid, $token);

// Get an object from its sid. If you do not have a sid,
// check out the list resource examples on this page
$feedback = $client
    ->account
    ->calls("CAe03b7cd806070d1f32bdb7f1046a41c0")
    ->feedback()
    ->create(
        3, array("issue" => "imperfect-audio")
    );

echo $feedback->dateCreated->format('Y-m-d H:i:s');

File: rest/call-feedback/instance-post-example-1/instance-post-example-1.5.x.cs
// Download the twilio-csharp library from twilio.com/docs/csharp/install
using System;
using Twilio;
using Twilio.Rest.Api.V2010.Account.Call;
using System.Collections.Generic;

class Example
{
    static void Main(string[] args)
    {
        // Find your Account Sid and Auth Token at twilio.com/console
        const string accountSid = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
        const string authToken = "your_auth_token";
        TwilioClient.Init(accountSid, authToken);

        var issues = new List<FeedbackResource.IssuesEnum>() {
            FeedbackResource.IssuesEnum.ImperfectAudio
        };
        FeedbackResource.Create("CAe03b7cd806070d1f32bdb7f1046a41c0",
                                3,
                                issue: issues);
    }
}
